pkgs/applications/editors/vscode/generic.nix

```diff
@@ -59,7 +59,7 @@ in
 
     buildInputs = (if stdenv.isDarwin
       then [ unzip ]
-      else [ gtk2 at-spi2-atk wrapGAppsHook ] ++ atomEnv.packages)
+      else [ gtk2 at-spi2-atk libdbusmenu-gtk3 wrapGAppsHook ] ++ atomEnv.packages)
```

Comment:
If it's only needed on `LD_LIBRARY_PATH` then this can be removed.

Reply:
Yes, only LD_LIB_P is needed, thanks. I removed that from the PR.